### PR TITLE
Added a message regarding moving forward with StaticArrays

### PR DESCRIPTION
**Pull request by andyferris**
@SimonDanisch @c42f I wasn't sure exactly how to word this... opinions?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -338,6 +338,14 @@ faster!
 
 ### *FixedSizeArrays* and *ImmutableArrays* compatibility
 
+Several existing packages for statically sized arrays have been developed for
+Julia, noteably *FixedSizeArrays* and *ImmutableArrays*. Upon consultation, it
+has been decided to move forward with *StaticArrays* which has found a new home
+in the *JuliaArrays* github organization. It is recommended that new users use
+this package, and that existing dependent packages consider switching to
+*StaticArrays* sometime during the life-cycle of Julia v0.5 as development for
```
Comment (c42f) [inline README.md]: Nitpick - I'd simplify this to

"sometime during the life cycle of Julia v0.5."

**Comment by c42f** [comment]
Looks fine to me.

**Comment by KristofferC** [comment]
Completely unrelated to this PR but I wanted to put it somewhere. Changing code to use StaticArrays always makes you happy:

ContMechTensors.jl: https://gist.github.com/KristofferC/949adf01633dee6a6837a7520de472f5
NearestNeighbors.jl: https://gist.github.com/KristofferC/d3666e6428a473ea93ccfea396391ab3

**Comment by andyferris** [comment]
@KristofferC do you understand the source of any of those regressions? Some things were twice as slow.

@SimonDanisch I'm going to merge this now. The plan is to tag a 0.1.0 version sometime soon and make an announcement to julia-users sometime soon.

**Pull request merged**